tests/system/marathon_common_tests.py

```diff
@@ -1154,3 +1157,173 @@ def test_ipv6_healthcheck(docker_ipv6_network_fixture):
         "The number of healthy tasks is {}, but {} was expected".format(app['tasksHealthy'], target_instances_count)
 
     client.remove_app(app['id'], True)
+
+@shakedown.dcos_1_11
+@pytest.mark.skipif("shakedown.ee_version() is None")
+def test_faultdomains_default():
```

Comment:
How about a `skipif("fault_domais < 2")`

Reply:
Again, I am not sure if we should silently ignore this set of tests, instead of making sure they run in the correct environment..

Reply:
We do so for multi cluster tests. Same for EE tests. However, I guess we could split up the tests and just run the once that are supposed to pass.

Reply:
Ps. That particular test `test_faultdomains_default` passes even without any regions defined. So there is no need for any particular pre-conditions.